tests/profiler/bug00360.phpt

```diff
@@ -22,7 +22,7 @@ cmd: %sbug00360.php
 part: 1
 positions: line
 
-events: Time_(µs) Memory_(bytes)
+events: Time_(10ns) Memory_(bytes)
```

Comment:
There should be at least one test where we check the range of the value too. I suggest modifying `tests/profiler/bug00714.phpt`, as that test some aspects of the values already.

Reply:
please push directly

Reply:
![image](https://user-images.githubusercontent.com/2228672/89509509-b1d5c380-d7cf-11ea-8e8c-237023a7b851.png)

should be allowed from my side